openslides/motions/migrations/0005_auto_20180202_1318.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2018-02-02 12:18
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Permission
+from django.db import migrations
+
+
+def delete_old_comment_permission(apps, schema_editor):
+    """
+    Deletes the old 'can_see_and_manage_comments' permission which is
```

Comment:
Ok so, but now nobody gets the new permission, so former admins are not able to handle comments any more.

Reply:
Done.